src/arraytraits.rs

```diff
@@ -323,7 +323,18 @@ where
     }
 }
 
-/// Implementation of ArrayView2::from(&S) where S is a slice to a 2D array
+/// Implementation of ArrayView2::from(&[[A; N]; M])
+///
+/// **Panics** if the product of non-zero axis lengths overflows `isize` (This can only occur if A
+/// is zero-sized because slices cannot contain more than `isize::MAX` number of bytes).
```

Comment:
This comment and the one for `ArrayViewMut` should mention that another panicking case is when `N == 0` and the number of rows is greater than `isize::MAX`, even if `A` is not zero-sized. (See #1131.)